packages/ms-report/src/__tests__/sequenceSVG.nucleotide.test.js

```diff
@@ -63,5 +63,5 @@ test('sequenceSVG of nucleotide', () => {
   let svg = sequenceSVG(sequence, info, options);
   writeFileSync(join(__dirname, 'testNucleotide.svg'), svg);
 
-  expect(svg).toHaveLength(27019);
+  expect(svg).toHaveLength(21219);
```

Comment:
The files are smaller because a lot of props like `svgjs:data="{&quot;leading&quot;:&quot;1.3&quot;}"` were removed with the update. You can see below that the SVG is visually the same, but I don't know if you relied on these data properties.

Reply:
Thanks. This was indeed more a regression test that anything else.